src/main/java/filter/expression/Qualifier.java

```diff
@@ -898,4 +969,53 @@ private void applyState(TurboIssue issue) throws QualifierApplicationException {
     public String getName() {
         return name;
     }
+
+    private Qualifier replaceMilestoneAliasQualifier(List<TurboMilestone> allMilestones, int currentIndex) {
+        String contents = content.get();
+        String alias = contents;
+
+        Pattern aliasPattern = Pattern.compile("(curr(?:ent)?)(?:(\\+|-)(\\d+))?");
+        Matcher aliasMatcher = aliasPattern.matcher(alias);
+        int offset = 0;
+
+        while (aliasMatcher.find()) {
+            if (aliasMatcher.group(2) != null && aliasMatcher.group(3) != null) {
+                offset = Integer.parseInt(aliasMatcher.group(3));
+                if (aliasMatcher.group(2).equals("+")) {
+                    currentIndex += offset;
+                } else {
+                    currentIndex -= offset;
+                }
+            } else if (!aliasMatcher.group(0).equalsIgnoreCase(alias)) {
+                // in this case found curr/current but there are something behind it
+                // e.g. curr+asdf
+                break;
+            }
+
+            if (currentIndex < allMilestones.size() &&
+                    currentIndex >= 0) {
+                contents = allMilestones.get(currentIndex).getTitle().toLowerCase();
+            }
+
+            break;
```

Comment:
Is this right? The loop will only execute once because of this. If it's intended the loop can be a guard clause instead, i.e `if (!matcher.find()) return this;`.